Client/components/MessageForm.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {Component} from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Link
+} from 'react-router-dom';
+import '../public/style.css';
+import $ from '../public/jquery.js';
+import AppActions from '../actions/AppActions';
+import ls from 'local-storage';
```

Comment:
Remove this if it's not being used. I doubt you need this as local storage is accessible from the client side